jetty-util/src/main/java/org/eclipse/jetty/util/StringUtil.java

```diff
@@ -376,12 +376,15 @@ public static void append(StringBuilder buf,
                               int offset,
                               int length)
     {
-        int end = offset + length;
-        for (int i = offset; i < end; i++)
+        synchronized (buf)
```

Comment:
Why is this synchronized now?
Wouldn't that mean all of our code using StringBuilder would now need synchronized now to make any use of this change?